llvm/lib/Target/EraVM/Disassembler/EraVMDisassembler.cpp

```diff
@@ -39,8 +39,53 @@ class EraVMDisassembler : public MCDisassembler {
                               ArrayRef<uint8_t> Bytes, uint64_t Address,
                               raw_ostream &CStream) const override;
 };
+
+Register decodeGR256(const MCDisassembler *D, unsigned EncodedReg) {
+  const auto *MCRI = D->getContext().getRegisterInfo();
+  const auto &RC = MCRI->getRegClass(EraVM::GR256RegClassID);
+  assert(RC.getNumRegs() == 16 && "Unexpected change to register class");
+  return RC.getRegister(EncodedReg);
+}
+
+int64_t decodeCC(unsigned EncodedCC) {
+  const unsigned EncodingForIfGTOrLT = 7;
```

Comment:
Don't we have it defined somewhere? Also if we plan to support `GTOrLT` it makes sense to add `TODO: <issue number>`.

Reply:
At now, I assume `IfGTOrLT` to be deprecated, please correct me if I'm wrong. Related: #440.

Reply:
Does is stand for `greater than or less than`? Why not name it `not equal` (`ne`)? Sounds more conventional.

Reply:
@hedgar2017 that why I proposed to rename lags to `x`, `y`, `z` to remove associations. `gt`, `lt`, `eq` is a lie. It's true for `sub` instruction only. Instruction set some flags on overflow, on division by zero, etc. So `ne` means that `eq` flag is not set for this particular instruction; `gl` means that `gt` or `lt` is set regardless of `ne`.

Reply:
Keeping this conversation unresolved until we decide on #440.